Runtime/Scripts/Schema/Sampler.cs

```diff
@@ -121,11 +122,30 @@ public enum WrapMode
             }
         }
 
-        public void Apply(Texture2D image) {
+        public void Apply(Texture2D image,
+                          MinFilterMode defaultMinFilter = MinFilterMode.NearestMipmapLinear,
+                          MagFilterMode defaultMagFilter = MagFilterMode.Linear) {
             if (image == null) return;
             image.wrapModeU = wrapU;
             image.wrapModeV = wrapV;
-            image.filterMode = filterMode;
+
+            // Use the default filtering mode for textures that have no such specification in data
```

Comment:
While not incorrect, this variant would be the same result with a little less code:

```csharp
image.filterMode = ConvertFilterMode(
    minFilter == MinFilterMode.None ? defaultMinFilter : minFilter,
    magFilter == MagFilterMode.None ? defaultMagFilter : magFilter
);
```

Please change if you agree